plugins/woocommerce/client/admin/client/settings-payments/components/payment-gateways.tsx

```diff
@@ -127,7 +128,15 @@ export const PaymentGateways = () => {
 				) }
 			</>
 		),
-		after: <Gridicon icon="chevron-right" />,
+		after: (
+			<a
+				href={ getAdminLink(
+					'admin.php?page=wc-settings&tab=checkout&section=offline'
+				) }
+			>
+				<Gridicon icon="chevron-right" />
```

Comment:
Not sure if this is in scope but vertical spacing on the chevron looks off

<img width="1516" alt="image" src="https://github.com/user-attachments/assets/c92fcc72-a07f-4951-a296-4ce7cfe65946">

Reply:
Thanks for reviewing, @psealock! This is out of scope, it should be addressed by the [draft PR](https://github.com/woocommerce/woocommerce/pull/52161) or follow-ups